src/main/java/nextstep/ladder/Player.java

```diff
@@ -0,0 +1,25 @@
+package nextstep.ladder;
+
+public class Player {
+    public static final int MAX_NAME_LENGTH = 5;
+    private String name;
```

Comment:
```suggestion
public class Player {
    public static final int MAX_NAME_LENGTH = 5;
    private final String name;
```
`Player` 잘 추가해주셧네요 👍 

name의 경우 불변하게 유지시켜줄 수 있을 것 같아요 😄